docs/.vitepress/ComponentsStatus.vue

```diff
@@ -0,0 +1,97 @@
+<script setup>
+import { computed } from 'vue';
+
+// props: 'released' | 'developing' | 'planned' | 'unsupported'
+const props = defineProps({
+  react: {
```

Comment:
Adding a validator to the props would give the consumer a hint of possible values. 🙂 
```
 validator: (value) => ['released', 'developing', 'planned', 'unsupported'].includes(value)
```

Reply:
perfect!